### PR TITLE
berkeleydb 5.3.28

### PR DESCRIPTION
**Pull request by Mno-hime**

### CODE REVIEW

File: components/database/berkeleydb/berkeleydb.p5m [hunk 1]
```diff
@@ -21,37 +21,22 @@
...
-set name=info.classification \
-    value=org.opensolaris.category.2008:System/Databases
+set name=info.classification value=org.opensolaris.category.2008:System/Databases
 set name=info.keyword value=bdb
-set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
-set name=org.opensolaris.arc-caseid value=PSARC/2010/300
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file usr/bin/db_archive path=usr/bin/$(MACH32)/db_archive
```
Comment (pyhalov) [inline components/database/berkeleydb/berkeleydb.p5m]: Do we ever need 32-bit version of the tools? If we don't, just deliver 64-bit versions. If we do, ship both and restore isaexec usage

Comment (Mno-hime) [reply components/database/berkeleydb/berkeleydb.p5m]: How do I know we need 32-bit version of the tools? There are certainly packages depending on BDB, not sure on what exactly. Restored `isaexec`.

File: components/database/berkeleydb/berkeleydb.p5m [hunk 2]
```diff
@@ -85,38 +86,30 @@ hardlink path=usr/bin/dbsql target=../lib/isaexec pkg.linted.userland.action002.
 file path=usr/include/db.h
 file path=usr/include/db_cxx.h
 file path=usr/include/dbsql.h
-link path=usr/lib/$(MACH64)/libdb-5.1.so target=./libdb-5.3.so
 file path=usr/lib/$(MACH64)/libdb-5.3.so
-link path=usr/lib/$(MACH64)/libdb-5.so target=./libdb-5.3.so
-link path=usr/lib/$(MACH64)/libdb.so target=./libdb-5.3.so
-link path=usr/lib/$(MACH64)/libdb_cxx-5.1.so target=./libdb_cxx-5.3.so
+link path=usr/lib/$(MACH64)/libdb-5.so target=libdb-5.3.so
+link path=usr/lib/$(MACH64)/libdb.so target=libdb-5.3.so
```
Comment (pyhalov) [inline components/database/berkeleydb/berkeleydb.p5m:91]: Please, preserve old -5.1 compatibility symlinks

Comment (Mno-hime) [reply components/database/berkeleydb/berkeleydb.p5m:91]: Done.

File: components/database/berkeleydb/berkeleydb.p5m [hunk 3]
```diff
@@ -21,22 +21,49 @@
...
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
-set name=org.opensolaris.arc-caseid value=PSARC/2010/300
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license berkeleydb.license license="berkeleydb license"
+file path=usr/docs/LICENSE
+
+# Keep v5.1 compat links
+link path=usr/lib/libdb-5.1.so target=libdb-5.3.so
```
Comment (pyhalov) [inline components/database/berkeleydb/berkeleydb.p5m]: Sorry, but why do we need them?

Comment (Mno-hime) [reply components/database/berkeleydb/berkeleydb.p5m]: I thought this is what you asked me to preserve when you said "preserve old -5.1 compatibility symlinks".

Comment (pyhalov) [reply components/database/berkeleydb/berkeleydb.p5m]: Yes, sure. I'm just not sure, if we need them or not.

Comment (pyhalov) [reply components/database/berkeleydb/berkeleydb.p5m]: But I miss something, you deliver not only 5.1-compat symlinks in this section, and some actions are included twice, like 'link path=usr/lib/libdb.so target=libdb-5.3.so'

**Pull request merged**